src/vmware/setup.py

```diff
@@ -8,7 +8,7 @@
 from io import open
 from setuptools import setup, find_packages
 
-VERSION = "5.0.3"
+VERSION = "6.0.0"
```

Comment:
```suggestion
VERSION = "5.1.0"
```
In fact, we only recommend upgrading the major version when the PR changes include breaking changes. If only some new features are released, it is recommended to upgrade the secondary version.